src/main/java/mate/dao/MyCoolResource.java

```diff
@@ -15,6 +15,10 @@ public static MyCoolResource openResource() {
         return new MyCoolResource(new ArrayList<>());
     }
 
+    public String getTime() {
```

Comment:
this method only returns first element that we added, it looks like it's redundant